app/app_test.go

```diff
@@ -56,8 +69,14 @@ func (at *appTest) reset() {
 	at.accOut = types.MakeAcc("output0")
 
 	eyesCli := eyes.NewLocalClient("", 0)
-	at.app = NewBasecoin(eyesCli)
-	at.app.SetLogger(log.TestingLogger().With("module", "app"))
+	// l := log.TestingLogger().With("module", "app"),
+	l := log.NewTMLogger(os.Stdout).With("module", "app")
```

Comment:
Let's try to use multi-character variable names generally (here `logger` maybe), Jae requested this, (I believe it's in to the coding_standards document) - once in a while it might makes sense to have single char variables for counters etc, but here let's expand